e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy-and-exports/summary-list/check-your-answers-policy-and-exports-multiple-different-name-summary-list.spec.js

```diff
@@ -0,0 +1,114 @@
+
+const {
+  ROOT: INSURANCE_ROOT,
+  CHECK_YOUR_ANSWERS: {
+    TYPE_OF_POLICY,
+  },
+} = ROUTES.INSURANCE;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      TYPE_OF_POLICY: { POLICY_TYPE },
+      CONTRACT_POLICY: {
+        REQUESTED_START_DATE,
+        CREDIT_PERIOD_WITH_BUYER,
+        POLICY_CURRENCY_CODE,
+        MULTIPLE: {
+          TOTAL_MONTHS_OF_COVER,
+          TOTAL_SALES_TO_BUYER,
+          MAXIMUM_BUYER_WILL_OWE,
+        },
+      },
+      ABOUT_GOODS_OR_SERVICES: { DESCRIPTION, FINAL_DESTINATION },
+      NAME_ON_POLICY: { NAME, POSITION },
+    },
+    ACCOUNT: { EMAIL },
+  },
+} = FIELD_IDS;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.submitApplication.tasks.checkAnswers;
+
+context('Insurance - Check your answers - Policy and exports - Multiple contract policy - Different name - Summary List', () => {
+  let url;
+  let referenceNumber;
+
+  before(() => {
+    cy.completeSignInAndGoToApplication({}).then(({ referenceNumber: refNumber }) => {
+      referenceNumber = refNumber;
+      cy.completePrepareApplicationMultiplePolicyType({ referenceNumber, differentPolicyContact: true });
+
+      task.link().click();
+
+      // To get past "Eligibility" check your answers page
+      cy.submitCheckYourAnswersForm();
+
+      url = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${TYPE_OF_POLICY}`;
+
+      cy.assertUrl(url);
+    });
+  });
+
+  beforeEach(() => {
+    cy.saveSession();
+
+    cy.navigateToUrl(url);
+  });
+
+  after(() => {
+    cy.deleteApplication(referenceNumber);
+  });
+
+  it(`should render a ${POLICY_TYPE} summary list row`, () => {
+    checkSummaryList.multipleContractPolicy[POLICY_TYPE]();
+  });
+
+  it(`should render a ${REQUESTED_START_DATE} summary list row`, () => {
+    checkSummaryList[REQUESTED_START_DATE]();
+  });
+
+  it(`should render a ${TOTAL_MONTHS_OF_COVER} summary list row`, () => {
+    checkSummaryList.multipleContractPolicy[TOTAL_MONTHS_OF_COVER]();
+  });
+
+  it(`should render a ${TOTAL_SALES_TO_BUYER} summary list row`, () => {
+    checkSummaryList.multipleContractPolicy[TOTAL_SALES_TO_BUYER]();
+  });
+
+  it(`should render a ${MAXIMUM_BUYER_WILL_OWE} summary list row`, () => {
+    checkSummaryList.multipleContractPolicy[MAXIMUM_BUYER_WILL_OWE]();
+  });
+
+  it(`should render a ${CREDIT_PERIOD_WITH_BUYER} summary list row`, () => {
+    checkSummaryList[CREDIT_PERIOD_WITH_BUYER]();
+  });
+
+  it(`should render a ${POLICY_CURRENCY_CODE} summary list row`, () => {
+    checkSummaryList[POLICY_CURRENCY_CODE]();
+  });
+
+  it(`should render a ${DESCRIPTION} summary list row`, () => {
+    checkSummaryList[DESCRIPTION]();
+  });
+
+  it(`should render a ${FINAL_DESTINATION} summary list row`, () => {
+    checkSummaryList[FINAL_DESTINATION]();
+  });
+
+  it(`should render a ${NAME} summary list row`, () => {
+    checkSummaryList[NAME](false);
```

Comment:
can we have an object structure for this new param so it has more context when in use, not sure what the `false` means here otherwise, can be e.g `({ sameName: false })`